### PR TITLE
Fix horizontal scrollbar bug in modals

### LINKED ISSUE #5032: UI bug horizontal scrollbar in modals
### Expected behavior
Modals should not have horizontal scroll.
I would suggest that we remove duplicate `overflow: auto` in children. Only one element should have `overflow-y: auto` for the main dialog scroll. 

### Actual behavior
image

### Steps to reproduce
This is probably triggered on different displays with different resolution, i use a 32 inch 4k display. To reproduce test by using different zoom settings in the browser

### PR DESCRIPTION
**Pull request by ikem-legend**
### What was the problem?

This PR resolves #5032

### How was it solved?

- Updating the styling for the different transaction modals

### How was it tested?

Visually

### CODE REVIEW

**Review by eniolam1000752 — CHANGES_REQUESTED** [review]
Should the Dialog component its self benefit from not having a horizontal scroll. 

This is so that other uses cased of the component don't suffer from an horizontal scroll.

**Comment by ikem-legend** [comment]
> Should the Dialog component its self benefit from not having a horizontal scroll.
> 
> This is so that other uses cased of the component don't suffer from an horizontal scroll.

I considered doing so but was uncertain about if that might possibly create problems in some other views

**Review by eniolam1000752 — DISMISSED** [review]

File: src/modules/pos/validator/components/RegisterValidatorView/index.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const RegisterValidator = ({ history, t }) => {
   }, []);
```
Comment (oskarleonard) [inline src/modules/pos/validator/components/RegisterValidatorView/index.js:15]: NRTTL found some more
image
image

I think we should do as we discussed in slack some time ago, this problem got introduced in https://github.com/LiskHQ/lisk-desktop/pull/4988 if my memory serves me correctly, which is caused by children inside children both having scroll overflow

**Review by eniolam1000752 — APPROVED** [review]

File: src/modules/pos/validator/components/RegisterValidatorView/index.js (hunk 1 above)
Comment (ikem-legend) [reply src/modules/pos/validator/components/RegisterValidatorView/index.js:15]: Fixed

**Review by ManuGowda — APPROVED** [review]
Tested on mac works fine (couldn't test unlock and claim rewards)

**Pull request merged**